data/stoa0040/stoa011/stoa0040.stoa011.opp-lat2.xml

```diff
@@ -61,7 +61,7 @@
                <pubPlace>Leipzig</pubPlace>
                   <date>1895-1911</date>
               </imprint>
-                <biblScope unit="volume">Sectio II, Pars 1-5</biblScope>
+                <biblScope unit="volume">Sectio II, Pars I-IV</biblScope>
```

Comment:
Latin language attribute

Reply:
Soooo close. Almost done. You have quite the eye @lcerrato